UnrealPlugin/Source/PlayfabGSDK/Public/ConnectedPlayer.h

```diff
@@ -5,12 +5,12 @@
 #include "CoreMinimal.h"
 
 #include "ConnectedPlayer.generated.h"
-
+FString PlayerId;
 USTRUCT(BlueprintType)
 struct FConnectedPlayer
 {
 	GENERATED_BODY()
 public:
-	UPROPERTY(BlueprintReadOnly)
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
```

Comment:
I agree with the change BlueprintReadOnly -> BlueprintReadWrite.

Any reason you are suggesting the EditAnywhere change?

Reply:
I just wanted to have accessibility both on C++ and Blueprint Graph.

Reply:
The EditAnywhere property isnt related to C++ afaik. In C++ you can access FStringId easily since FConnectedPlayer is a struct